### DISCUSSION TITLE
How to reference a tag in the text of a document

### DISCUSSION THREAD

**Discussion by mogplus88** [vkbo/novelWriter#1861] [Help]
I"ve been using NovelWriter for a while now and it is proving very useful and generally easy to use. Tags, I find, are particularly useful. I understand how tags are used to reference an entry in one of the preset folders (plot, character, location, etc) but I can't figure out  how to reference a tag specified in a chapter, scene or note of the novel. I can define a tag there, but how do I reference it from elsewhere? e.g.
```
#How to write a best seller
## Use NovelWriter
@tag: UseNovelwriter
...
## Writing Tools
@ref: UseNovelWriter
```
What should "@ref" be?

Thanks.

**Comment by vkbo** [comment]
There is no keyword for that. The intention of the system is to reference notes on story elements, not the other way around. I was considering adding the feature, but there has been no demand for it.

There's no technical reason for it, it was just never added.

**Comment by vkbo** [comment]
Feel free to make a feature request for it if you want. It's not a problem adding it. Just need a good suggestion for a keyword. Not as generic as `@ref` perhaps.

**Reply by mogplus88** [reply]
Hi Veronica,
Thanks for the reply. I don't think it's worth a feature request. When I was playing around with tags I just thought I must have missed something, as there was the ability to create a tag but no way to reference it. Even if the functionality was there I don't know if I would actually use it! 
As for a keyword, since it's a reference to a link in a "novel" folder, maybe @novel? Or @main, as it's in the main text? ( should appeal to a python programmer :) ) Or @book?  @text?
;-) Ian

**Reply by vkbo** [reply]
Please don't tag random people on GitHub with `@` tags when you're describing novelWriter notation. Use the back tick symbol to wrap them.

**Reply by vkbo** [reply]
Anyway, thanks for the suggestions. I May add this anyway at some point. If not for anything other than completeness. `@novel` may make the most sense.
